io/input/router.go

```diff
@@ -464,6 +468,10 @@ func (q *Router) processEvent(e event.Event, system bool) {
 		cstate, evts := q.cqueue.Push(state.clipboardState, e)
 		state.clipboardState = cstate
 		q.changeState(e, state, evts)
+	case transfer.URLEvent:
+		var evts []taggedEvent
+		evts = append(evts, taggedEvent{tag: q, event: e})
```

Comment:
Like `key.Event` above, I would not expect a `transfer.URLEvent` to have an associated tag.